astropy/coordinates/baseframe.py

```diff
@@ -810,11 +810,13 @@ def represent_as(self, new_representation, in_frame_units=False):
         >>> from astropy.coordinates import SkyCoord, CartesianRepresentation
         >>> coord = SkyCoord(0*u.deg, 0*u.deg)
         >>> coord.represent_as(CartesianRepresentation)
-        <CartesianRepresentation x=1.0, y=0.0, z=0.0>
+        <CartesianRepresentation (x, y, z) in
```

Comment:
Yea, I'm in favor of just dropping the "in" -- I could tack that on to this PR? @eteq ?